lib/updateVersion.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
```

Comment:
There is a bug in `underscoreApplicationSpecificSemantics` that isn't directly related to this PR but is probably worth fixing now.

From 1.0 to 2.0, `WEIGHT` should map to `WEIGHTS_0` and `JOINT` should map to `JOINTS_0`.

Instead what I'm seeing for CesiumMan is starting as:

```
"JOINT": "accessor_115",
"NORMAL": "accessor_25",
"POSITION": "accessor_23",
"TEXCOORD_0": "accessor_27",
"WEIGHT": "accessor_112"
```

and ending as

```
"JOINT": 2,
"NORMAL": 5,
"POSITION": 4,
"TEXCOORD_0": 6,
"_WEIGHT": 1
```

but it should be:

```
"JOINTS_0": 2,
"NORMAL": 5,
"POSITION": 4,
"TEXCOORD_0": 6,
"WEIGHTS_0": 1
```